Binary files wxorig/src/common/.DS_Store and wxnew/src/common/.DS_Store differ
diff -uNr wxorig/src/common/log.cpp wxnew/src/common/log.cpp
--- wxorig/src/common/log.cpp	2010-11-05 08:58:46.000000000 -0400
+++ wxnew/src/common/log.cpp	2011-03-02 21:37:44.000000000 -0500
@@ -685,7 +685,7 @@
                     ms_timestamp, wxLocaltime_r(&timeNow, &tm));
 
         str->Empty();
-        *str << buf << wxS(": ");
+        *str << buf << wxS(":\t");
     }
 #endif // wxUSE_DATETIME
 }
